schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "FOCAL2PIXEL_Y": {
+           "description": "Transformation y coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "SENSOR_LOCATION": {
+           "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+      },
+       "SENSOR_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+	     },
+       "SUN_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
+           "type": "array",
+           "items": {
+               "type": "array",
+               "items" : {
+                  "type": "number",
+                  "minItems": 4 ,
+                  "maxItems": 4
+           }
+          }
+      },
+
+       "DETECTOR_LINE_RATE": {
+           "description": "The line rates of the detector map with the associated start line and time",
+           "type": "array",
+              "items": {
+	               "type": "object",
+	                "properties": {
+	                   "START_LINE": {
+		                     "type": "number"
+	                    },
+                      "LINE_START_TIME": {
+ 	                        "type": "number"
+	                     },
+	                     "EXPOSURE_TIME": {
+		                       "type": "number"
+	                     },
+	          "required": [ "EXPOSURE_TIME"]
+	                 }
+	              }
+      },
+       "DETECTOR_SAMPLE_SUMMING": {
+           "description": "The summing in the sample-direction",
```

Comment:
We should specify that summing of 1 is no summing, summing of 2 is 2 pixel summing, 4 is 4 pixel summing, etc. Not all missions use this format.  For some missions, they use 0 is no summing, 1 is 2 pixel, 2 is 4 pixel, etc.

Reply:
:+1: